mods/ra/sequences/structures.yaml

```diff
@@ -131,12 +131,16 @@ powr:
 
 apwr:
 	idle:
+		Offset: 0,-10
```

Comment:
You mention in one of the other commits that you are also adjusting for future footprint positions, so I guess this may be intentional.